packages/api-explorer/src/JsonForm/configureJsonEditor.js

```diff
@@ -12,7 +12,17 @@ function setDefaultCustomization (JSONEditor) {
     JSONEditor.defaults.editors[key] = class Customization extends JSONEditor.defaults.editors[key] {
       showEditJSON(){
         super.showEditJSON()
-        if (this.editjson_holder && this.editjson_textarea && this.editjson_control) {
+        if (this.schema.id === 'root' && this.editor_holder && this.editjson_card_holder && this.editjson_holder && this.editjson_textarea) {
+          this.editjson_holder.style.display = 'none'
+          this.editor_holder.style.display = 'none'
+          this.editjson_card_holder.style.display = 'block'
+          this.editjson_textarea.style.width = `100%`
+          this.editjson_textarea.style.height = '500px'
+          this.editjson_textarea.style.resize = 'vertical'
+          this.editjson_textarea.style.fontFamily = 'monospace'
+        } 
+
+        if (this.schema.id === undefined && this.editjson_holder && this.editjson_textarea && this.editjson_control) {
```

Comment:
I'd not check against `this.schema.id === undefined`.

I'd check on `!== root` or add a return statement in the previous `if`.

Reply:
Done!